### PR TITLE
Expose new s= URL parameter in MWMPoint.

### PR DESCRIPTION
**Pull request by kw217**
Now that style support has been added to MAPS.ME (https://github.com/mapsme/omim/pull/335), this PR adds it to the Android API. Resolves https://github.com/mapswithme/api-android/issues/13.

### CODE REVIEW

**Comment by kw217** [comment]
I got a suggestion from @rsudev on https://github.com/cgeo/cgeo/issues/5239:

> Regarding the maps.me Api: As the possible styles seem to be fixed in code, wouldn't it make sense to expose them as an enumeration in the Api?

Would that be good? I'm happy to change the PR.

**Comment by kw217** [comment]
@deathbaba Do you have any thoughts on @rsudev's suggestion? I'm happy either way.

**Comment by kw217** [comment]
Hi @d-kunin @deathbaba @Bananeweizen - please can someone review this PR and merge it or let me know what needs to happen next? I'm keen to start using this change in an Android app. Thanks.

File: lib/src/com/mapswithme/maps/api/MWMPoint.java [hunk 1]
```diff
@@ -63,10 +74,20 @@ public MWMPoint(double lat, double lon, String name, String id)
...
 
+  /**
+   * Sets the style (appearance) for this point. Null, empty-string or unrecognized appears as a
+   * violet circle. Styles known to MAPS.ME appear with the appropriate symbol. At the time of
+   * writing, supported styles are "placemark-red", "placemark-blue", "placemark-purple",
+   * "placemark-yellow", "placemark-pink", "placemark-brown", "placemark-green", "placemark-orange",
+   * all of which are displayed as a small flag of the indicated colour.
+   * @param style
+   */
+  public void setStyle(String style) { mId = style; }
```
Comment (yunikkk) [inline lib/src/com/mapswithme/maps/api/MWMPoint.java]: mStyle = style, not mId

Comment (d-kunin) [inline lib/src/com/mapswithme/maps/api/MWMPoint.java]: I would say that you probably want to make an enum for style to make it strongly-typed and typo-prof.

File: lib/src/com/mapswithme/maps/api/MWMPoint.java [hunk 2]
```diff
@@ -63,10 +74,20 @@ public MWMPoint(double lat, double lon, String name, String id)
...
+
   @Override
   public String toString()
   {
-    return "MWMPoint [lat=" + mLat + ", lon=" + mLon + ", name=" + mName + ", id=" + mId + "]";
+    return "MWMPoint [lat=" + mLat + ", lon=" + mLon + ", name=" + mName + ", id=" + mId + ", style=" + mStyle + "]";
```
Comment (d-kunin) [inline lib/src/com/mapswithme/maps/api/MWMPoint.java]: It is a matter of one's taste, but making this line multi-line will make code a bit more readable.

**Comment by kw217** [comment]
Thanks - I've applied the above feedback: used an enum, and split long line.

**Comment by d-kunin** [comment]
@kw217 nice! unfortunately I don't have access rights to merge it, but  @deathbaba and @yunikkk should be able to

**Comment by yunikkk** [comment]
Seems good, LGTM

**Comment by deathbaba** [comment]
@d-kunin now you have write permission
@Zhdanovich you're added too into iOS api repo )

**Comment by kw217** [comment]
Cool, thanks!

**Pull request merged**